src/server/pegasus_server_impl.cpp

```diff
@@ -675,12 +697,19 @@ void pegasus_server_impl::on_multi_get(const ::dsn::apps::multi_get_request &req
         return;
     }
 
-    int32_t max_kv_count = request.max_kv_count > 0 ? request.max_kv_count : INT_MAX;
+    int32_t max_iteration_count =
+        (request.max_kv_count > 0 && request.max_kv_count < _rocksdb_max_iteration_count)
+            ? request.max_kv_count
+            : _rocksdb_max_iteration_count;
     int32_t max_kv_size = request.max_kv_size > 0 ? request.max_kv_size : INT_MAX;
+    int32_t max_iteration_size_config =
+        _multi_get_max_iteration_size > 0 ? _multi_get_max_iteration_size : INT_MAX;
+    int32_t max_iteration_size = std::min(max_kv_size, max_iteration_size_config);
```

Comment:
Different style with L700~L703, I'd prefer the style here.

Reply:
I update L700~L701 code style:
```
uint32_t max_kv_count = request.max_kv_count > 0 ? request.max_kv_count : INT_MAX;
uint32_t max_iteration_count = std::min(max_kv_count, _rocksdb_max_iteration_count);
```